fetch.bs

```diff
@@ -2366,6 +2366,7 @@ known as an <dfn export id=concept-aborted-network-error>aborted network error</
 <p>A <a>network error</a> is a <a for=/>response</a> whose
 <a for=response>status</a> is always 0,
 <a for=response>status message</a> is always the empty byte sequence,
+<a for=response>body info</a> is always a new <a for=/>response body info</a>,
 <a for=response>header list</a> is always empty, and
 <a for=response>body</a> is always null.
```

Comment:
Nit: can we put body info at the end throughout as it's defined last of these fields?